Scripts/github/prepare-simulators.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -l
```

Comment:
This is _very_ similar to a file with the same name in Valet. Thank you @NickEntin for figuring out this absurdity.

Reply:
Haha these two lines took me so long to figure out:

```
.../Xcode_11.7.app/.../iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime ...
.../Xcode_10.3.app/.../iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime ...
```

Because, you know, why not mix things up